build.ts

```diff
@@ -32,11 +31,18 @@ const accents = [
 
 async function makeThemeObject(
   name: string,
-  accent: keyof Labels<Color, AlphaColor>,
-  palette: Labels<Color, AlphaColor>
+  accent: string,
+  flavor: CatppuccinFlavor,
 ) {
+  let themeAccent: string = "";
+  flavor.colorEntries.map(([colorName, { hex }]) => {
+    if (colorName == accent) {
+      themeAccent = hex;
+    }
+  });
+  const palette = flavor.colors;
```

Comment:
I don't know why we really need to change from `palette[accent].hex` since it seems more clean and clear to me, but if we do this map and everything is totally unnecessary.
```suggestion
  const palette = flavor.colors;
  const themeAccent = palette[accent].hex;
```